### ISSUE TITLE
Is there a way to generate RSS feed for blog posts?

### ISSUE THREAD

**Issue by mabasic** [themsaid/katana#9]

**Comment by themsaid** [comment]
Will be working on that next week as I got many requests regarding this matter.

**Comment by mabasic** [comment]
Great stuff. I'm currently converting my personal website and blog to katana. I'll send you a link if you want for reference.

**Comment by themsaid** [comment]
Would be interested to check it it out :+1: thanks :)

**Comment by mabasic** [comment]
It's online now. Thank you for this great tool @themsaid 

http://mariobasic.com/

**Comment by themsaid** [comment]
Woo-Hooo :)
Looking good

**Comment by dailysleaze** [comment]
What i've done is create a normal blade file which sorts the blogs in reverse order of date and shows the latest 10 https://gist.github.com/dailysleaze/13897eff30e54ee787ca3406779ddf59

**Comment by themsaid** [comment]
@dailysleaze but it'll be converted to an html file instead of `.xml` then.

**Comment by dailysleaze** [comment]
True, but it the browser seems to detect it as a feed if it's using the content rather than the mime type. Agree it's not perfect.
